### ISSUE TITLE
Outdated docs and NPE when formatting table headers

### ISSUE THREAD

**Issue by tbnguyen1407** [alajmo/sake#42]
- [x] I have the latest version of sake
- [x] I have searched through the existing issues

## Info

- OS
  - [x] Linux
  - [ ] Mac OS X
  - [ ] Windows
  - [ ] other

- Shell
  - [x] Bash
  - [ ] Zsh
  - [ ] Fish
  - [ ] Powershell
  - [ ] other

- Version: 0.12.1

## Problem / Steps to reproduce

Seems the config reference doc is not updated. I ran into problem configuring header format in newer version.

https://github.com/alajmo/sake/blob/main/docs/config-reference.md?plain=1#L117

```
~ sake.yaml (old) ~
specs:
  default:
    output: table
themes:
  default:
    table:
      format:
        header: upper

~ sake.yaml (new) ~
specs:
  default:
    output: table
themes:
  default:
    table:
      header:
        format: upper
```

```
$ sake check           ## runs fine
$ sake list servers    ## error
```

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x657177]

goroutine 1 [running]:
github.com/alajmo/sake/core/print.CreateTable({{0x0, 0x0}, {0x7d28f6, 0x5}, {{0xc0000ada90, 0x7}, {{0x0, 0x0}, {0x7d2761, 0x5}, ...}, ...}, ...}, ...)
        /home/runner/work/sake/sake/core/print/table.go:31 +0x197
github.com/alajmo/sake/core/print.table1[...]({_, _, _}, {{0x0, 0x0}, {0x7d28f6, 0x5}, {{0xc0000ada90, 0x7}, {{0x0, ...}, ...}, ...}, ...}, ...)
        /home/runner/work/sake/sake/core/print/print_table.go:48 +0x88
github.com/alajmo/sake/core/print.PrintTable[...]({_, _, _}, {{0x0, 0x0}, {0x7d28f6, 0x5}, {{0xc0000ada90, 0x7}, {{0x0, ...}, ...}, ...}, ...}, ...)
        /home/runner/work/sake/sake/core/print/print_table.go:33 +0x19e
github.com/alajmo/sake/cmd.listServers(0xae2340, {0xb1bb18?, 0x0?, 0x0?}, 0xc0000d2fa0, 0xc000080780)
        /home/runner/work/sake/sake/cmd/list_servers.go:121 +0x5e9
github.com/alajmo/sake/cmd.listServersCmd.func1(0xc00015cc80?, {0xb1bb18?, 0x0?, 0x0?})
        /home/runner/work/sake/sake/cmd/list_servers.go:33 +0x88
github.com/spf13/cobra.(*Command).execute(0xc00015cc80, {0xb1bb18, 0x0, 0x0})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:876 +0x67b
github.com/spf13/cobra.(*Command).ExecuteC(0xadd680)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:990 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.5.0/command.go:918
github.com/alajmo/sake/cmd.Execute()
        /home/runner/work/sake/sake/cmd/root.go:42 +0x25
main.main()
        /home/runner/work/sake/sake/main.go:8 +0x17
```

**Comment by alajmo** [comment]
Thanks for finding this, I've updated sakecli.com config reference (the manpage had the correct but I forgot to update the site as well) and will fix the parse error to something prettier.

**Comment by alajmo** [comment]
Should be fixed now with v0.13.0, https://github.com/alajmo/sake/releases.
